README.md

```diff
@@ -3,42 +3,50 @@
 This repo serves to backup my neovim config file. I ended up making this public
 again for convenience.
 
-## Installation.
-Get Vim Plug (https://github.com/junegunn/vim-plug)
+## Installation
+
+First install Neovim. For Ubuntu, run the following:
+```
+sudo add-apt-repository ppa:neovim-ppa/stable
+sudo apt-get install -y neovim
+```
+
+Next, get Vim Plug (https://github.com/junegunn/vim-plug) by running the following:
 ```
 sh -c 'curl -fLo "${XDG_DATA_HOME:-$HOME/.local/share}"/nvim/site/autoload/plug.vim --create-dirs \
        https://raw.githubusercontent.com/junegunn/vim-plug/master/plug.vim'
 ```
-Afterwards, simply copy and paste the init.vim file into `~/.config/nvim/init.vim`.
+Afterwards, make a `nvim` directory under `~/.config/` and copy and paste the `init.vim` file from this repository into `~/.config/nvim/init.vim`.
```

Comment:
Include the method to make a directory (mkdir -p ~/.config/nvim)

Reply:
Done